packages/client/src/components/TextBox/demo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        '& > *': {
+            margin: theme.spacing(1),
+            width: '25ch',
+        },
+    },
+}));
+
+export default function TextFields(props) {
+    const classes = useStyles();
+
+    return (
+        <form className={classes.root} noValidate autoComplete="off">
+            {/*[type is either standard-basic + label is the name of the text field]*/}
+            <TextField id={props.type} label={props.label} />
```

Comment:
I think `props.type` here should go in `variant` actually, something like

`<TextField variant={props.type} label={props.label} />`

Reply:
Sure. I can do that, but how do I amend the new commit to this PR. I am assuming that force pushing is frowned upon.

Reply:
you can just make another commit on your branch and push again, the PR will update automatically! 

Also I do force push to sometimes for small changes but only on my own branches, never to branches that other people also commit to